client/src/modules/users/UserCashBoxManagementModal.html

```diff
@@ -0,0 +1,39 @@
+<form name="UserForm" bh-submit="UsersCashBoxModalCtrl.submit(UserForm)" novalidate>
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="title">
+        <span translate>USERS.UPDATING_USER</span>
+        <label class="badge badge-warning" translate>USERS.SET_CASHBOX</label>
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body" style="overflow-y: scroll; max-height:600px; ">
+
+
+    <div class="form-group" ng-class="{ 'has-error' : UserForm.$submitted && UserForm.display_name.$invalid }">
+      <label class="control-label col-md-12" translate>FORM.LABELS.USERNAME</label>
+      <div class="col-md-12">
+        <h4><i> {{ UsersCashBoxModalCtrl.user.display_name }} </i></h4>
+      </div>
+    </div>
+
+    <span class="help-block" translate> FORM.INFO.USER_CASHBOX </span>
```

Comment:
I think `help-block`s often go within the `form-group`.  Could you move this up into the form group?